lib/src/Base/Func/SpecFunc/SpecFunc.cxx

```diff
@@ -503,6 +509,42 @@ Scalar SpecFunc::DiLog(const Scalar x)
   return value;
 }
 
+// Exponential
+Scalar SpecFunc::Exp(const Scalar x)
+{
+#ifdef OPENTURNS_HAVE_FMATH
+  return fmath::expd(x);
+#else
+  return std::exp(x);
+#endif
+}
+
+Point SpecFunc::Exp(const Point & x)
+{
+  const UnsignedInteger size = x.getSize();
+#ifdef OPENTURNS_HAVE_FMATH
+  Point result(size);
+  for (UnsignedInteger i = 0; i < size; ++i)
+    result[i] = fmath::expd(x[i]);
+  return result;
+  /*  if (size == 0) return Point(0);
+  std::vector< double > result(x.begin(), x.end());
+  if (size % 2 == 0)
```

Comment:
maybe n should be a multiple of 4 instead of 2 when avx is used

Reply:
@jschueller In fmath.hpp, lines 536-537, the documentation of fmath::expd_v reads:
```
	px : pointer to array of double
	n : size of array(assume multiple of 2 or 4)
```